src/view/dataElements/xdmObject.jsx

```diff
@@ -115,22 +142,32 @@ const XdmObject = ({
               <FieldLabel labelFor="schemaField" label="Schema" />
             </InfoTipLayout>
           </div>
-          <div>
-            <Select
+          <div className="u-flex">
+            <ComboBox
```

Comment:
In Slack you mentioned:

> There is a small issue when filtering the schema results and deleting the filter. After removing all the text search query, the combo box results do not update unless you click twice on the button to drop down the list.

I wanted to make sure you're aware that the bug might be a little more complex than this. Notice when I type "s" and then backspace and repeat the process:

![searchconcurrency](https://user-images.githubusercontent.com/210820/97040930-f39f1c00-152b-11eb-9368-9e76947581eb.gif)

Notice the results vary.

Reply:
Here's some other weird behavior. Type "s", then backspace, then "u", then backspace.

![filteringproblem2](https://user-images.githubusercontent.com/210820/97190147-fcc9fc00-176a-11eb-8285-9585566547dc.gif)

Reply:
I think the problem here is that the React-Spectrum ComboBox already has filtering in it (when you type in the input, ComboBox will filter down the options to just those that match), but we're also doing filtering on our side as the user types and then updating the `options` passed into ComboBox. ComboBox is getting all kinds of confused.

I'm not sure, but I think continuing with ComboBox will lead us down a long rabbit hole. I would recommend swapping it out for something like https://git.corp.adobe.com/reactor/extension-adobeanalytics/blob/master/src/view/extensionConfiguration/components/restrictedComboBox.jsx, which leverages Autocomplete but makes it look similar to the ComboBox component. Autocomplete has a `getCompletions` API, which is much more suitable for this type of use case since we won't be competing with the component on who's doing the filtering.

Reply:
CompletionComboBox from the analytics extension is closer to what we need here. I am going to try and see if that fixes the issues.